### PR TITLE
Add pattern for gls like commands

### PR DESCRIPTION
**Pull request by hesstobi**
I use the \gls command almost everywhere. It would be nice if the command has grammar scope to trigger auto completion and ignore the labels for spell checking.

### CODE REVIEW

File: grammars/latex.cson [hunk 1]
```diff
@@ -1920,6 +1920,35 @@
       }
     ]
   }
+  {
+    'begin': '((\\\\)(?:(?:gls|GLS|Gls)\\w*\\*?))(?:(\\[)[^\\]]*(\\]))?(\\{)'
```
Comment (Aerijo) [inline grammars/latex.cson]: This regex will match anything that starts with `gls`, even including stuff like `gls1234` which is invalid. The scope needs to be heavily reduced, either to a specific list of possible matches, or one like [this](https://github.com/hesstobi/language-latex/blob/2e3aaa68fa9beba51ad058332b85ef16e7ec4119/grammars/latex.cson#L1831)

**Comment by hesstobi** [comment]
OK, I updated the PR with a more specific regex. I also updated the change log.

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,7 @@ based on http://keepachangelog.com/en/1.0.0/
 
 ### Added
 - Basic spec for the LaTeX grammar.
+- Grammar for `gls` and `glstext` like commands
```
Comment (yudai-nkt) [inline CHANGELOG.md]: Could you add a period at the end of the sentence to make our changelog consistent? I'm afraid we don't document how to document a changelog anywhere, but I'll be very happy if you'd agree to change.

Comment (hesstobi) [reply CHANGELOG.md]: Yes of course.

**Comment by Aerijo** [comment]
All requested changes made, so I'm merging.

**Pull request merged**